config/application.rb

```diff
@@ -9,7 +9,7 @@
 module Points
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 7.0
```

Comment:
Aside note: if we were moving from `config.load_defaults 5.1` to `config.load_defaults 6.0` we could decide if go with the default behavior which adds `zeitwerk` of disable `zeitwerk` adding the following both lines:

```
config.load_defaults 6.0
config.autoloader = :classic
```
but since we moved right to `config.load_defaults 7.0` there are no more options regarding `autoloading`, `zeitwerk` is a must.